modules/twinklexfd.js

```diff
@@ -258,12 +259,31 @@ Twinkle.xfd.callback.change_category = function twinklexfdCallbackChangeCategory
 			work_area = work_area.render();
 			old_area.parentNode.replaceChild(work_area, old_area);
 
-			$(work_area).find('[name=delsort]')
-				.attr('data-placeholder', 'Select delsort pages')
-				.chosen({width: '100%'});
+			if ($.fn.select2) {
+				$(work_area).find('[name=delsort]')
+					.attr('data-placeholder', 'Select delsort pages')
```

Comment:
So one kind of annoying thing is that apparently already selected items aren't removed/disabled from the dropdown if they're already in the multiselect pillbox.  I imagine there's a jquery-ish way to toggle an item disabled<->enabled/hidden<->visible when it moves in or out of the select, but unless it's straightforward, I doubt it's worth it here.

Reply:
Adding `'.select2-results__options [aria-selected=true] { display: none; }'` to the CSS does the trick. 

Clicking on the already selected item is another way to de-select it.  (Unlike in chosen where a single backspace deletes an option, here de-selecting an option can't be done by a single backspace - that just converts it to the text which then has to removed.) Would that have some utility? I rather guess not.

Reply:
The issue is if you're typing/using the keyboard to navigate and then hit enter, you get... nothing.  It's a weird situation, as least in my mind, to have the result polluted with non-op options, especially when they're already visible.  The CSS isn't sufficient since you'll still get "phantom" results (like the optgroup...) if it's the only result and the menu behaves like it's there.  I think the answer is "meh" and we can all live.

Reply:
The CSS is hacky, yeah. Using jquery to toggle items hidden/visible also amounts to the same thing, and disabling/enabling doesn't seem to make sense as these are `li`s, not `option`s. So we have to use remove the items outright. This is something for which someone will need to do the dirty work of finding out how exactly which event to listen to, and/or what select2 config option to use. I can't say I'm volunteering.